config/initializers/ab_tests.rb

```diff
@@ -30,4 +30,15 @@ module AbTests
           0,
     },
   ).freeze
+
+  IDV_TEN_DIGIT_OTP = AbTestBucket.new(
+    experiment_name: '10-digit OTP for IdV',
+    default_bucket: :six_alphanumeric_otp,
+    buckets: {
+      ten_digit_otp:
+        IdentityConfig.store.ab_testing_idv_ten_digit_otp_enabled ?
+          IdentityConfig.store.ab_testing_idv_ten_digit_otp_percent :
+          0,
```

Comment:
IMO it would be clearer just to have one config, the percent? because zero percent change is the same as disabling it?
```suggestion
        IdentityConfig.store.ab_testing_idv_ten_digit_otp_percent,
```